scripts/build.sh

```diff
@@ -118,7 +118,14 @@ fi
 # Build k-NN lib and plugin through gradle tasks
 cd $work_dir
 # Gradle build is used here to replace gradle assemble due to build will also call cmake and make before generating jars
-./gradlew build --no-daemon --refresh-dependencies -x integTest -DskipTests=true -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
+./gradlew build --no-daemon --refresh-dependencies -x integTest -x test -Dopensearch.version=$VERSION -Dbuild.snapshot=$SNAPSHOT -Dbuild.version_qualifier=$QUALIFIER
+./gradlew :buildJniLib -Dsimd.enabled=false
```

Comment:
So the default gradle build will build _avx2, and this call will build the non-avx2 on x86?
And for arm64 do we just have neon to be always on?

Reply:
Seems like gradle build now does not build the lib by default.

Reply:
yes, correct